### PR TITLE
noetic-devel

### PR DESCRIPTION
**Pull request by MatthijsBurgh**

### CODE REVIEW

File: ed_sensor_integration/CMakeLists.txt [hunk 1]
```diff
@@ -1,8 +1,8 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.0.2)
 project(ed_sensor_integration)
 
-## Compile as C++11, supported in ROS Kinetic and newer
-add_compile_options(-std=c++11)
+## Compile as C++14, supported in ROS Melodic and newer
+add_compile_options(-std=c++14)
```
Comment (Timple) [inline ed_sensor_integration/CMakeLists.txt]: Just curious. But isn't this the default in Melodic? And thus no need to set it explicitly?

Comment (MatthijsBurgh) [reply ed_sensor_integration/CMakeLists.txt]: When I run ` g++ -dM -E -x c++  /dev/null | grep -F __cplusplus` on Ubuntu 18.04 it prints: `#define __cplusplus 201402L`. So I think indeed C++14 is the default in Ubuntu 18.04. @jlunenburg Could you run this on Ubuntu 16.04?

Comment (jlunenburg) [reply ed_sensor_integration/CMakeLists.txt]: The athome18 laptop with 16.04 shows `#define __cplusplus 199711L` so I guess it's indeed not necessary to set this

File: ed_sensor_integration_msgs/package.xml [hunk 2]
```diff
@@ -10,12 +13,12 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
-  <build_depend>geometry_msgs</build_depend>
+  <depend>geometry_msgs</depend>
+  <depend>std_msgs</depend>
```
Comment (jlunenburg) [inline ed_sensor_integration_msgs/package.xml:17]: Do we need std_msgs?

Comment (MatthijsBurgh) [reply ed_sensor_integration_msgs/package.xml:17]: As discussed it is the also in all common msg pkgs

File: ed_sensor_integration/CMakeLists.txt (hunk 1 above)
Comment (MatthijsBurgh) [reply ed_sensor_integration/CMakeLists.txt]: We are removing it everywhere

**Pull request merged**